doc/development/plugins.rst

```diff
@@ -432,19 +432,17 @@ should have the form ``pluginname.devicename``, using periods for delimitation.
 
 PennyLane uses a setuptools ``entry_points`` approach to plugin discovery/integration.
 In order to make the devices of your plugin accessible to PennyLane, simply provide the
-following keyword argument to the ``setup()`` function in your ``setup.py`` file:
+following keyword argument to the ``[options.entry_points]`` section in your ``setup.cfg`` file:
```

Comment:
@maliasadi it might be best to keep reference to both `setup.py` and `setup.cfg` approaches here, for external plugin developers who might still be using `setup.py`.